pkgs/os-specific/linux/lvm2/common.nix

```diff
@@ -26,7 +27,13 @@ stdenv.mkDerivation rec {
   };
 
   nativeBuildInputs = [ pkg-config ];
-  buildInputs = [ udev libuuid libaio ];
+  buildInputs = lib.optionals udevSupport [
+    udev
+  ] ++ lib.optionals (!onlyLib) [
+    libuuid
+  ] ++ [
+    libaio
+  ];
```

Comment:
I think it's clearer if the unconditional input (`libaio`) is at the start of `buildInputs` rather than the end.

Reply:
Done